bind_test.go

```diff
@@ -217,17 +217,17 @@ func TestBindXML(t *testing.T) {
 }
 
 func TestBindForm(t *testing.T) {
-	assert := assert.New(t)
+	assertion := assert.New(t)
```

Comment:
Instead of using `assert.New(t)` refactor all asserts to use `assert.Equal(t, .......)` variants.